### PR TITLE
Update feature-flags.mdx with new environment variables

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier Lopez <javier@okteto.com>

### CODE REVIEW

**Review by codyjlandstrom — APPROVED** [review]

File: src/content/reference/feature-flags.mdx [hunk 1]
```diff
@@ -13,18 +13,21 @@ Admin variables are equivalent to defining that variable on every developer's ma
...
+| OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED           | Compose services mounting the same volume will be placed on the same node using Kubernetes's pod affinity.                                                                                                                      | `true`                 |
+| OKTETO_KUBERNETES_TIMEOUT                        | Specifies the timeout while deploying.                                                                                                                                                                                          | `0`                    |
+| OKTETO_FORCE_REMOTE                              | If true, the CLI will run deploy and destroy commands on the cluster instead of running them locally.                                                                                                                           | `false`                |
```
Comment (pchico83) [inline src/content/reference/feature-flags.mdx:21]: can we remove this one? it's gone as part of exposing the deploy on remote behavior from the admin ui

File: src/content/reference/feature-flags.mdx [hunk 2]
```diff
@@ -13,18 +13,21 @@ Admin variables are equivalent to defining that variable on every developer's ma
...
+| OKTETO_BUILDKIT_WAIT_TIMEOUT                     | Specifies the maximum duration to wait for BuildKit to become available.                                                                                                                                                        | `10m`                  |
+| OKTETO_RETRY_BUILDKIT_INTERVAL                   | Specifies the interval between successive checks of BuildKit's availability status.                                                                                                                                             | `5s`                   |
+| OKTETO_MAX_RETRIES_FOR_BUILDKIT_TRANSIENT_ERRORS | Specifies the maximum number of retry attempts for build processes that fail due to transient (temporary) BuildKit errors.                                                                                                      | `3`                    |
```
Comment (pchico83) [inline src/content/reference/feature-flags.mdx:33]: can we document `OKTETO_BUILDKIT_FRONTEND_IMAGE`?

File: src/content/reference/feature-flags.mdx [hunk 3]
```diff
@@ -13,18 +13,21 @@ Admin variables are equivalent to defining that variable on every developer's ma
...
+| OKTETO_SYNCTHING_VERSION                         | Specifies the Syncthing version the CLI must use.                                                                                                                                                                               | N/A                    |
+| OKTETO_BUILDKIT_WAIT_TIMEOUT                     | Specifies the maximum duration to wait for BuildKit to become available.                                                                                                                                                        | `10m`                  |
+| OKTETO_RETRY_BUILDKIT_INTERVAL                   | Specifies the interval between successive checks of BuildKit's availability status.                                                                                                                                             | `5s`                   |
```
Comment (pchico83) [inline src/content/reference/feature-flags.mdx:32]: If we have time, I would rename this to `OKTETO_BUILDKIT_RETRY_INTERVAL`, that way buildkit variables are together is everything is sorted alphabetically

File: src/content/reference/feature-flags.mdx (hunk 2 above)
Comment (pchico83) [inline src/content/reference/feature-flags.mdx:33]: If we have time, I would rename this to `OKTETO_BUILDKIT_MAX_RETRIES_FOR_TRANSIENT_ERRORS`, that way buildkit variables are together is everything is sorted alphabetically

File: src/content/reference/feature-flags.mdx [hunk 4]
```diff
@@ -13,18 +13,21 @@ Admin variables are equivalent to defining that variable on every developer's ma
...
\ No newline at end of file
+| Environment Variable                             | Description                                                                                                                                                                                                                     | Default Value          |
+|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
+| OKTETO_SMART_BUILDS_ENABLED                      | Enables the [Smart Builds feature](core/build-service.mdx#smart-builds) that uses a hash of the build context to determine if an image needs to be rebuilt or can be pulled from the registry. This can accelerate build times. | `true`                 |
```
Comment (pchico83) [inline src/content/reference/feature-flags.mdx:18]: can we sort variables in alphabetical order?

**Review by pchico83 — APPROVED** [review]

File: src/content/reference/feature-flags.mdx (hunk 3 above)
Comment (codyjlandstrom) [reply src/content/reference/feature-flags.mdx:32]: Agree with this. That would make it much easier to keep these sorted in the documentation

**Pull request merged**